packages/integrations/vercel/src/serverless/adapter.ts

```diff
@@ -59,11 +59,10 @@ const ISR_PATH = `/_isr?${ASTRO_PATH_PARAM}=$0`;
 // https://vercel.com/docs/concepts/functions/serverless-functions/runtimes/node-js#node.js-version
 const SUPPORTED_NODE_VERSIONS: Record<
 	string,
-	{ status: 'current' } | { status: 'beta' } | { status: 'deprecated'; removal: Date }
+	{ status: 'default' } | { status: 'beta' } | { status: 'retiring'; removal: Date | string; warnDate: Date } | { status: 'deprecated'; removal: Date }
 > = {
-	16: { status: 'deprecated', removal: new Date('February 6 2024') },
-	18: { status: 'current' },
-	20: { status: 'beta' },
+	18: { status: 'retiring', removal: "Early 2025", warnDate: new Date('October 1 2024') },
```

Comment:
nit: can we use ISO dates? e.g. `2024-10-1`

Reply:
counternit 😆: I would have to think for a second whether 10 is the month or the day.